src/minuteman_metrics.erl

```diff
@@ -377,8 +375,8 @@ process_p99s_4_test_() ->
 process_dst_ip_map_test_() ->
     DAddr = {54, 192, 147, 29},
     IP = 16#36c0931d,
-    Conn = {ip_vs_conn, tcp, established, 167792566, 47808, IP, 8080, 167792566, 8081, 59},
-    Conn2 = {ip_vs_conn, tcp, established, 167792567, 47808, IP, 8080, 167792566, 8081, 59},
+    Conn = {ip_vs_conn, tcp, established, 167792566, 47808, 167792566, 8080, IP, 8081, 59},
```

Comment:
Erlang docs usually say to never use tuple representation for records because the underlying implementation may one day change. However, a lot of existing applications uses tuples to manipulate records, and they have always been tuples, so I don't think it will change any time soon.